python/django_bolt/middleware/compiler.py

```
@@ -425,6 +432,44 @@ def add_optimization_flags_to_metadata(metadata: dict[str, Any] | None, handler_
+
+    Raises:
+        ImproperlyConfigured: An entry is not an address or a CIDR block.
+    """
+    configured = getattr(settings, "BOLT_TRUSTED_PROXIES", None)
+    # Check the raw value. Normalizing first would turn an empty string into an
+    # empty list, and a typo that empties the setting must not read as "no
+    # proxies declared".
+    if isinstance(configured, str):
+        raise ImproperlyConfigured(
+            f"BOLT_TRUSTED_PROXIES must be a list of addresses or CIDR blocks, got the string {configured!r}."
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)